charts/src/lib/plugins/example/index.js

```diff
@@ -0,0 +1,31 @@
+import MomentumCharts from '../../index.js';
+
+const example = () => {
+
+  MomentumCharts.Plugins.install(MomentumCharts.Board, 'export', function () {
+    console.log(this.Svg);
```

Comment:
Where is this.Svg coming from?

Reply:
from MomentumCharts.Board where the plugin sets up